components/armDecoder/encodings/armLoadStore.cpp

```diff
@@ -493,7 +497,7 @@ arminst LDP(armcode const &aFetchedOpcode, uint32_t aCPU, int64_t aSequenceNo) {
   simple_action act = addAddressCompute(inst, addr_deps);
   addReadXRegister(inst, 1, rn, addr_deps[0], true);
 
-  if (index != kSignedOffset) {
+  if (index == kPreIndex || index == kPostIndex) {
```

Comment:
This is a good change but can we document it a bit better?

Reply:
Ok, I will add the reference from the manual.